wpilibcExamples/src/main/cpp/examples/StateSpaceElevator/cpp/Robot.cpp

```diff
@@ -109,7 +109,7 @@ class Robot : public frc::TimedRobot {
     // Sets the target height of our elevator. This is similar to setting the
     // setpoint of a PID controller.
     frc::TrapezoidProfile<units::meters>::State goal;
-    if (m_joystick.GetRightBumper()) {
```

Comment:
Should the Java examples make the same update?

Reply:
No, they use `Joystick`. It's probably a good idea to make them consistent though.